app/components/dashboard/hypertension/overdue_patients_called_component.html.erb

```diff
@@ -0,0 +1,48 @@
+<div id="overdue-patients-called-trend" class="d-lg-flex w-lg-50 pl-lg-2">
+  <%= render Dashboard::Card::GraphComponent.new(id: "overdueCalledTrend", data: graph_data, period: @period) do |c| %>
+    <%= c.title(title: "Overdue patients called".html_safe, subtitle: t("overdue_patients_called.reports_card_subtitle")) do |title| %>
+        <%= title.tooltip({ "Numerator" => t("overdue_patients_called.numerator"),
+                            "note" => t("overdue_patients_called.numerator_note"),
+                            "divider" => "",
+                            "Denominator" => t("overdue_patients_called.denominator", region_name: @region.name) }) %>
+      <%end%>
+      <%= c.summary do %>
+        <div class="mb-12px d-lg-flex align-lg-center">
+          <p class="c-print-black graph-percent fs-28px" data-key="overduePatientsCalledRate" data-format="percentage"  style="color: #e0b000;"></p>
+          <div>
+            <p class="m-0px c-black">
+              <span data-key="overduePatientsCalled" data-format="numberWithCommas"></span>
+              patients called in
+              <span data-key="endDate"></span>
+            </p>
+            <p class="m-0px c-grey-dark c-print-black">
+              of
+              <span data-key="overduePatients" data-format="numberWithCommas"></span>
+              patients that are overdue at the start of
+              <span data-key="endDate"></span>
+            </p>
+          </div>
+        </div>
+        <div class="pb-8px">
+          <p class="fw-bold mb-4px">
+            Call results <span data-key="endDate"></span>
+          </p>
+          <div style="white-space: nowrap; display: flex; flex-direction: row; grid-gap: 8px;">
+            <p data-title="chartProportionalPercentageCalledWithResultAgreedToVisit"
+                style="border-radius: 30px; text-align: center; width: 33%; margin: 0; padding: 2px 8px; overflow: hidden; color: rgba(44, 115, 0, 1); background: rgba(78, 206, 0, 0.15);">
+              <span data-key="chartProportionalPercentageCalledWithResultAgreedToVisit"></span>%
+              'Agreed to visit'
+            </p>
+            <p data-title="chartProportionalPercentageCalledWithResultRemindToCallLater"
+              style="border-radius: 30px; text-align: center; background: rgba(252, 247, 193, 0.8); color:rgb(118, 93, 3); margin: 0; white-space: nowrap;  width: 33%; margin: 0; padding: 2px 8px; overflow: hidden;">
+              <span data-key="chartProportionalPercentageCalledWithResultRemindToCallLater"></span>% 'Remind to call'
```

Comment:
Are the quotes around the text required? These would appear on the dashboard as well.

Reply:
We can remove these quotes